src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -243,6 +262,7 @@ class BasicSettings extends React.Component {
         changes.operatingSystemId = this.props.defaultValues.operatingSystemId
         changes.memory = this.props.defaultValues.memory
         changes.cpus = this.props.defaultValues.cpus
+        changes.topology = this.getTopologySettings(changes.cpus)
```

Comment:
This does not really matter because `this.props.defaultValues.cpus` will always get the value `1` so both `this.getTopologySettings` and `this.props.defaultValues.topology` will come to the same result. But yeah, I can change that, just for consistency with the rest of the values here which are set to default, of course 😏

Reply:
Assuming that the default/blank template is unchanged, then yes. But...the default template can be changed!

Reply:
👍🏽 Yes, it can (but I don't think it ever happens) :)